devel/libjwt/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libjwt
+version             1.8.0
+revision            0
+categories          devel
+platforms           darwin
+license             MIT
+
+maintainers         nomaintainer
+
+description         C library for Javascript Web Tokens (JWT's)
+long_description    ${description}
+
+homepage            https://github.com/benmcollins/libjwt
+master_sites        https://github.com/benmcollins/libjwt/archive/
```

Comment:
If this is on GitHub, you probably want to use the GitHub portgroup. See the guide for details.

Reply:
Yes, the following setup line should work, but I have not tested it:

`github.setup            benmcollins libjwt 1.8.0 v`

You will also need to update the checksums and and add the `size`.